finality-aleph/src/party/mod.rs

```diff
@@ -501,6 +435,20 @@ where
                 }
             }
         }
+
+        loop {
```

Comment:
Maybe a comment here why it is not an infinite loop would be good. But also as far as I understand now it is possible to start a session without knowing an authorities for it and such behavior won't end up as `panic`

Reply:
I will add a comment here, only the first ever session can have such behaviour. Also this loop is here to  prevents from advancing to session start without knowing the authorities

Reply:
Added

Reply:
actually we no longer have a loop here, now we are waiting for a notification here